cockroachcloud/cmek.md

```diff
@@ -0,0 +1,125 @@
+
+When you enable CMEK on a {{ site.data.products.dedicated }} cluster, {{ site.data.products.db }} creates two encryption keys and begins to use them to protect newly-written data at rest. {{ site.data.products.db }} manages these encryption keys and propagates them to cluster nodes.
+
+1. The _data key_ is a Data Encryption Key (DEK), and is used to encrypt and decrypt cluster data before it is read from or written to disks attached to a cluster's nodes. Each time the cluster is started or restarted, and each time a node and related disks are added to a cluster, {{ site.data.products.dedicated }} uses the store key to encrypt and decrypt data keys. Each cluster node maintains its own list of data keys. The data key is automatically rotated monthly and is always encrypted at rest by the store key.
+
+1. The _store key_ is a Key Encryption Key (KEK) that encrypts the data keys at rest. It is encrypted at rest by the CMEK key before being propagated to cluster nodes, and is decrypted in memory on each node when the cluster starts. The store key is not automatically rotated. 
+
+For more details about encryption in {{ site.data.products.db }}, see [Encryption At Rest](/docs/{{site.versions["stable"]}}/encryption.html).
+
+Enabling CMEK has no impact on a cluster's existing data.
+
+{{site.data.alerts.callout_success}}
+CMEK is configured per cluster region. A single-region cluster is actually a multi-region cluster with only a single region. The following steps occur for each of a cluster's regions.
+{{site.data.alerts.end}}
+
+At the time that you enable CMEK for a cluster, {{ site.data.products.db }}:
+
+1. Creates the store key and encrypts it using the CMEK key.
+1. Creates the data key and encrypts it using the encrypted store key.
+1. Propagates the store key and data key to cluster nodes.
+1. Starts the cluster.
+
+Going forward:
+
+1. The cluster can start only when the CMEK key is available to decrypt the store key. If the CMEK key becomes unavailable or you revoke the cluster's access to the key, the cluster can't start until access is restored.
+1. Each time the store key is rotated, the new store key is also encrypted using the CMEK key.
+1. Each time the data key is rotated, the new data key is encrypted using the encrypted store key.
+1. Each time a node writes new data to disk, it is encrypted using the current data key. Data is read using the data key that was used to encrypt it.
+{{ site.data.products.db }}
+{{site.data.alerts.callout_danger}}
+If the CMEK key is destroyed, the cluster's data can't be recovered or restored from a managed backup in {{ site.data.products.db }} or from a manual backup to the same cluster. It may be possible to restore a manual backup to a new cluster. 
+{{site.data.alerts.end}}
+
+## Backing up and restoring to a cluster with CMEK
```

Comment:
```suggestion
## Backup and restore operations on a cluster with CMEK
```
Minor suggestion here, but I prefer something like this because the current heading and its gerunds imply that the section will explain _how_ to do it, whereas this is conceptual only.